regression/aws-ec2-tests/build-default-afis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Again, please DRY this stuff out with CI machinery and put it in python. Or if CI is sufficiently comprehensive, can you not just delete of this code altogether?

Reply:
CI can't run this since building the AFIs will take more than 6hrs... so how are you envisioning DRYing things?

Reply:
Oh interesting... since I switched to using a dynamically self-hosted runner we can now run jobs longer than 6 hours (if we use the VM that GitHub provides its bottlenecked to 6hrs). See https://github.community/t/is-that-possible-to-run-job-which-takes-more-than-6-hours-on-self-hosted-runner/17121/3. This should allow up to run things like `buildafi` (and stretch goal of SPEC(N)) using CI! Though I'll let someone else implement that exersize.